GOVERNANCE.md

```diff
@@ -0,0 +1,221 @@
+- In these cases, if no consensus can be reached, the matter may be resolved by Simple Majority Vote <https://en.wikipedia.org/wiki/Majority> by the appropriate team.
+- However there are cases that always require a Simple Majority or Supermajority Vote <https://en.wikipedia.org/wiki/Supermajority> by the appropriate team.
+These are defined below.
+
+### Simple Majority Changes
+
+- Org team: Licensing and intellectual property changes
+- Org team: Using Flux/CNCF money for anything CNCF deems "not cheap and easy"
+
+### Supermajority Changes
+
+- Org team: Enforcing a Code of Conduct violation.
+- Org team: Removing a member from a team for any reason other than inactivity.
+- Org team: Material changes to the Governance document.
+  - Editorial changes to governance may be made by Org team lazy consensus, unless challenged.
+These are changes which fix spelling or grammar, update work affiliation or similar, update style or reflect an outside and obvious reality.
+They do not change the intention or meaning of anything in this document.
+
+### Membership Changes
+
+- All team members MUST remain active contributors.
+If they are unresponsive for > 3 months they will be automatically removed unless a supermajority of the other members of that team agrees to extend the period.
+- A member may voluntarily step down from a team by following the proposal process in the community repo.
+These proposals are automatically accepted.
+Once offboarding is complete and the issue is closed, they become an emeritus member of that team.
+- New members may be added to a team by a supermajority vote by the existing members of that team.
+- When a team has no members the Org team become responsible for it and may either archive the team or find new members.
+- When a team is archived, its members become emeritus members of that team.
+
+Org team membership changes have additional considerations:
+
+- Members of any other team are eligible to be nominiated as an Org team member
+- Org maintainers MUST remain current members of other non-archived teams.
+If that status changes, they will also loose Org team membership.
+- If an Org maintainer volunteraily steps down, in addition to the process above, within 7 calendar days the Flux dev list MUST be notified <https://lists.cncf.io/g/cncf-flux-community>.
+This gives contributors reasonable time to be made aware of the change.
+- When there is an opening for a new Org maintainer, any contributor to a repository in the `fluxcd` github org may nominate a suitable existing member of another team as a replacement.
+  - The nomination period will be three weeks starting the day after an Org team member opening becomes available.
+  - The nomination must be made as a reply to the notification topic in the Flux dev list.
+
+## Licenses
+
+- Apache 2.0 is required for all source code repositories
+- Developer Certificate of Origin (DCO) commit signoff is required for all new code contributions
```

Comment:
Just to be clear, this does not refer to PGP signing of commits, but only to DCO git "--signoff￼" on commits.

Reply:
PS, this browser extension has been useful for helm team members and other people, to add DCO signoff when making commits from the GitHub UI￼ (with her fast PRs that are made UI￼-only, or for committing GitHub PR review "suggestions￼", etc. https://github.com/scottrigby/dco-gh-ui

Reply:
I am not referring to PGP signatures, but the sign-off using `git commit -s`. See for example the commit history on the `fluxcd/flux` repository as an example that we do not enforce it at present: https://github.com/fluxcd/flux/commits/master

Reply:
Please let’s not enforce sign-off on commits, it’s not only completely useless in terms of identity proof  but also a major pain for new contributors when we request them to rebase PRs.

Reply:
Just to clarify @stefanprodan:

```
       -s, --signoff
           Add Signed-off-by line by the committer at the end of the commit log message. The meaning of a signoff depends on the project, but it typically certifies that committer has the rights to submit this work under
           the same license and agrees to a Developer Certificate of Origin (see http://developercertificate.org/ for more information).
```

I do not see how this would give conflicts while rebasing?

Reply:
I’ve seen it many times, when you resolve conflicts, if you keep the last commit message then you’ll be removing the previous signoff. What’s the point of this anyway, without pgp anyone can impersonate another user by just signing off with that user’s email.

Reply:
> What’s the point of this anyway, without pgp anyone can impersonate another user by just signing off with that user’s email.

It is not about impersonation, but about authorship vs copyright acknowledgement: https://stackoverflow.com/a/1962112

Reply:
Ok